### ISSUE TITLE
[11.0] pdf.js can't print PDF files on Google Chrome

### ISSUE THREAD

**Issue by Linekio** [odoo/odoo#24996]
Impacted versions:
11.0
Steps to reproduce:
With Google Chrome, go to Discuss, and send a message with a PDF file as an attachment.
Click on the PDF file, and try to print it (without downloading it)
Current behavior:
You can't print the PDF. With Ctrl+P, you got an error  "Printing is not fully supported by this browser"
Expected behavior:
There should be a print icon
Video/Screenshot link (optional):
![capture du 2018-05-30 10-22-24](https://user-images.githubusercontent.com/25034353/40782754-a2d844de-64e0-11e8-9bb2-361a857d1ae3.png)

It may be enough to update pdf.js ?

Since I send a lot of pdf files on discuss (and all my users using Chrome), I let the browser use its own viewer :
```
<?xml version="1.0" encoding="UTF-8"?>
<templates>

    <t t-extend="DocumentViewer.Content">
        <t t-jquery="iframe" t-operation="replace">
            <iframe class="mb48 o_viewer_pdf" t-if="widget.activeAttachment.type == 'application/pdf'" t-attf-src="/web/content/#{widget.activeAttachment.id}" />
        </t>
    </t>

</templates>
```

**Comment by mart-e** [comment]
@ged-odoo a recent version of pdf.js does not seem to have this problem. Can we consider upgrading the lib?
cf demo https://mozilla.github.io/pdf.js/web/viewer.html
I am not sure which version we have ourself but I wouldn't be surprise we still use the one back from website_slide merge.

**Comment by reyelts** [comment]
+1 requesting solution to this problem.

**Comment by Linekio** [comment]
Fixed with #49457
